locales/sv.yml

```diff
@@ -0,0 +1,65 @@
+swe:
+    start:
+        success: Strömmen startades framgångsrikt
+        error: "Fel kan inte starta strömmen: %{error}"
+    startTwitchTranscoding:
+        trying: Försöker starta strömmen med omkodning
+        successNoTranscoding: Strömmen startades framgångsrikt utan omkodning
+        success: Startade ström med omkodning %{attemptsMessage}
+        attempts: ", took %{count} attempts"
```

Comment:
Could you please translate this as well.

Reply:
> Could you please translate this as well.

Some of it is not directly translatable but ill get on it :)